lib/index.js

```diff
@@ -16,13 +16,15 @@ module.exports = function (testResults) {
 
     var failedMessage = '<!here> Bad News! *' + testResults.numFailedTestSuites + '* test suites have failed and *' + testResults.numFailedTests + '* tests have failed :( *' + testResults.numPassedTests + '* tests have passed. Please have a look at your tests.';
     var passedMessage = '<!here> Great News! All *' + testResults.numPassedTests + '* tests have passed! Code coverage is coming soon..';
-    var text = testResults.numFailedTestSuites > 0 || testResults.numFailedTests > 0 ? failedMessage : passedMessage;
+    var allTestPassed = testResults.numFailedTestSuites > 0 || testResults.numFailedTests > 0;
+    var text = allTestPassed ? failedMessage : passedMessage;
```

Comment:
I would rename the variable `allTestPassed` to `failedTests` since the value is true if there are failed tests:
```
var failedTests = testResults.numFailedTestSuites > 0 || testResults.numFailedTests > 0;
var text = failedTests ? failedMessage : passedMessage;
```

Reply:
also, do it in the `index.js` and not the `lib/index.js` file then build the package again.